Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+yapf = "*"
+prospector = "*"
+awsebcli = "*"
+pre-commit = "*"
+coverage = "*"
+
+[packages]
+Django = "*"
```

Comment:
pipenv is nice, it lets you specify your prod and dev dependencies, and handles creating the virtualenv and managing the versions of your dependencies' dependencies